packages/monorepo-scripts/src/deps_versions.ts

```diff
@@ -17,11 +17,31 @@ interface VersionsByDependency {
     [depName: string]: Versions;
 }
 
-const PACKAGE_JSON_GLOB = '../*/package.json';
+interface ParsedDependencies {
+    ignored: VersionsByDependency;
+    included: VersionsByDependency;
+}
+
+const PACKAGE_JSON_GLOB = '../../*/package.json';
+const config = JSON.parse(fs.readFileSync(path.join(__dirname, '../../../package.json')).toString()).config;
```

Comment:
```suggestion

const config = JSON.parse(fs.readFileSync(path.join(__dirname, '../../../package.json')).toString()).config;
```

Let's add a new line between top-of-file constants and the script itself.